components/Transaction/TransactionDetails.js

```diff
@@ -61,54 +39,61 @@ export const TransactionDetails = ({ tx }) => {
-                and validated in the ledger{" "}
-                <LinkLedger version={tx.outcome.ledgerVersion} />{" "}
-                (index:{tx.outcome.indexInLedger}).
-              </span>
-            )
-            : (
-              <span>
-                The transaction <b className="red">FAILED</b>{" "}
-                and included to the ledger{" "}
-                <LinkLedger version={tx.outcome.ledgerVersion} /> (index:{" "}
-                {tx.outcome.indexInLedger}).
-              </span>
-            )}
+          {tx.id} <CopyButton text={tx.id}></CopyButton>
         </Text>
-        <Table>
-          <Row>
-            <Cell>{t("table.type")}:</Cell>
-            <Cell>{tx.type}</Cell>
-          </Row>
-          <Row>
-            <Cell>Time (UTC):</Cell>
-            <Cell>{formatDateTime(tx.outcome.timestamp)}</Cell>
-          </Row>
-          <Row>
-            <Cell>Initiated by:</Cell>
-            <Cell>
-              <LinkAccount hash={tx.address} />
-            </Cell>
-          </Row>
-          <Row>
-            <Cell>Sequence:</Cell>
-            <Cell>#{tx.sequence}</Cell>
-          </Row>
-          <Row>
-            <Cell>XRLP fee:</Cell>
-            <Cell>{tx.outcome?.fee}</Cell>
-          </Row>
-          <Row>
-            <Cell>CTID:</Cell>
-            <Cell>{tx.rawTransaction?.ctid}</Cell>
-          </Row>
-        </Table>
+        {isSuccessful
+          ? (
+            <Text>
+              The transaction was <b className="green">successfull</b>{" "}
+              and validated in the ledger{" "}
+              <LinkLedger version={tx.outcome.ledgerVersion} />{" "}
+              (index:{tx.outcome.indexInLedger}).
+            </Text>
+          )
+          : (
+            <Text>
+              The transaction <b className="red">FAILED</b>{" "}
+              and included to the ledger{" "}
+              <LinkLedger version={tx.outcome.ledgerVersion} /> (index:{" "}
+              {tx.outcome.indexInLedger}).
+            </Text>
+          )}
+        <Table.Root>
+          <Table.Body>
+            <Table.Row>
+              <Table.Data>{t("table.type")}:</Table.Data>
+              <Table.Data>{tx.type}</Table.Data>
+            </Table.Row>
+            <Table.Row>
+              <Table.Data>Time:</Table.Data>
+              <Table.Data>
+                {formatDateTime(tx.outcome.timestamp)}
+              </Table.Data>
+            </Table.Row>
+            <Table.Row>
+              <Table.Data>Initiated by:</Table.Data>
+              <Table.Data>
+                <LinkAccount address={tx.address} />
+              </Table.Data>
+            </Table.Row>
+            <Table.Row>
+              <Table.Data>Sequence:</Table.Data>
+              <Table.Data>#{tx.sequence}</Table.Data>
+            </Table.Row>
+            <Table.Row>
+              <Table.Data>XRLP fee:</Table.Data>
+              <Table.Data>{tx.outcome?.fee}</Table.Data>
```

Comment:
```
import {amountFormat } from '../utils/format'

{amountFormat(tx.outcome?.fee)}
```

Reply:
`amountFormat` gives `1.2e-11 XRP` for low values, it looks a bit not user friendly.